pkg/mappings/mappings.go

```diff
@@ -116,13 +132,22 @@ func GenerateMappings(cmd *cobra.Command, _ []string) error {
 				scoping = meta.RESTScopeNameNamespace
 			}
 
+			slices.Sort(res.Verbs)
+
+			verbs := res.Verbs
+
+			if res.Verbs.String() == fmt.Sprintf("%v", DefaultVerbs) {
```

Comment:
Parsing to a string isn't necessary since they're both sorted.

```suggestion
			if slices.Equal(res.Verbs, defaultVerbs) {
```

Reply:
Good point! This was a remnant from before I realized sorting it would be very useful.